README.md

```diff
@@ -1,17 +1,228 @@
+
+> Note: This capabality is primarily targeted for experimentation and transitioning 
+to Pulumi. Pulumi's [desired state model][how-pulumi-works] greatly benefits
+from having resources be directly defined in your Pulumi program as demonstrated
+in the [workload example][workload-example].
+
+```typescript
+import * as eks from "@pulumi/eks";
+import * as k8s from "@pulumi/kubernetes";
+
+// Create an EKS cluster.
+const cluster = new eks.Cluster("my-cluster");
+
+// Deploy Wordpress into our cluster.
+const wordpress = new k8s.helm.v2.Chart("wordpress", {
+    repo: "stable",
+    chart: "wordpress",
+    values: {
+        wordpressBlogName: "My Cool Kubernetes Blog!",
+    },
+}, { providers: { "kubernetes": cluster.provider } });
+
+// Export the cluster's kubeconfig.
+export const kubeconfig = cluster.kubeconfig;
+```
+
+### Deploying a Workload using the Resource API
+
+This example creates a EKS cluster with [`pulumi/eks`](https://github.com/pulumi/pulumi-eks),
+and then deploys an NGINX Deployment and Service using the SDK resource API, and the 
+`kubeconfig` credentials from the cluster's [Pulumi provider](https://www.pulumi.com/docs/reference/programming-model/#providers).
+
+```typescript
+import * as eks from "@pulumi/eks";
+import * as k8s from "@pulumi/kubernetes";
+
+// Create an EKS cluster with the default configuration.
+const cluster = new eks.Cluster("my-cluster");
+
+// Create a NGINX Deployment and Service.
+const appName = "my-app";
+const appLabels = { appClass: appName };
+const deployment = new k8s.apps.v1.Deployment(`${appName}-dep`, {
+    metadata: { labels: appLabels },
+    spec: {
+        replicas: 2,
+        selector: { matchLabels: appLabels },
+        template: {
+            metadata: { labels: appLabels },
+            spec: {
+                containers: [{
+                    name: appName,
+                    image: "nginx",
+                    ports: [{ name: "http", containerPort: 80 }]
+                }],
+            }
+        }
+    },
+}, { provider: cluster.provider });
+
+const service = new k8s.core.v1.Service(`${appName}-svc`, {
+    metadata: { labels: appLabels },
+    spec: {
+        type: "LoadBalancer",
+        ports: [{ port: 80, targetPort: "http" }],
+        selector: appLabels,
+    },
+}, { provider: cluster.provider });
+
+// Export the URL for the load balanced service.
+export const url = service.status.loadBalancer.ingress[0].hostname;
+
+// Export the cluster's kubeconfig.
+export const kubeconfig = cluster.kubeconfig;
+```
+
+## Contributing
+
+If you are interested in contributing, please see the [contributing docs][contributing].
+
+## Code of Conduct
+
+You can read the code of conduct [here][code-of-conduct].
+
+[pulumi-kubernetes]: https://github.com/pulumi/pulumi-kubernetes
+[contributing]: CONTRIBUTING.md
+[code-of-conduct]: CODE-OF-CONDUCT.md
+[workload-example]: #deploying-a-workload-on-aws-eks
+[how-pulumi-works]: https://www.pulumi.com/docs/reference/how/
```

Comment:
This will also change to `https://www.pulumi.com/docs/intro/concepts/how-pulumi-works`. Sorry for the inconvenience!